components/list-steps/list-steps-en.html

```diff
@@ -587,3 +588,56 @@ <h4>Example with heading and section of step following a step started at 9, expe
 		</li>
 	</ol>
 </section>
+<section>
+	<h2 id="limitwidth">Limited width content</h2>
+	<p>List items are normally restricted in character width when using the class <code>cnt-wdth-lmtd</code>. A list step that contain a <code>&lt;div&gt;</code>, <code>&lt;section&gt;</code> or <code>&lt;table&gt;</code> tag will not be restricted in character width, nor the components within. </p>
+	<p>However, elements normally restricted by the <code>cnt-wdth-lmtd</code> class will still be effected within those tags.</p>
+	<section class="cnt-wdth-lmtd">
+		<h3>With cnt-wdth-lmtd class</h3>
+		<ol class="lst-stps ld-zr">
+			<li>
+				<h5>Header for step 1</h5>
+				<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent dictum urna justo, porta ultrices purus blandit a. Nunc sed quam dolor. Donec vitae dapibus turpis, in ultrices nisl. Phasellus et tristique diam. Sed a sem eget nisi vehicula tristique et sit amet elit. Vivamus porta turpis sed sapien posuere blandit. Phasellus faucibus quis augue hendrerit tincidunt.</p>
+			</li>
+			<li>
+				<h5>Header for step 2</h5>
+				<p>Duis condimentum volutpat blandit. Praesent non justo venenatis, efficitur ex et, commodo libero.</p>
+				<div class="row">
+					<div class="col-md-6">
+						<h6>Sub header for step 2</h6>
+						<p>Aliquam gravida, lorem at finibus pulvinar, orci neque dignissim erat, eget ornare velit sem tincidunt lectus. Duis condimentum volutpat blandit.</p>
+					</div>
+					<div class="col-md-6">
+						<h6>Sub header for step 2</h6>
+						<p>Phasellus non lacinia dolor, eu scelerisque est. Donec sed elementum odio. Sed molestie molestie tempor. Praesent non justo venenatis, efficitur ex et, commodo libero.</p>
+					</div>
+				</div>
+			</li>
+			<li>
+				<h5>Header for step 3</h5>
+				<p>Mauris ut consequat urna, sit amet interdum risus. Sed eu nulla ac nisl commodo laoreet nec eget ante.</p>
+				<details>
+					<summary>Example</summary>
+					<p>Quisque lorem tortor, cursus et pretium at, porta in risus. Donec ut placerat lorem. Curabitur neque ex, rutrum a justo sit amet, placerat viverra magna.</p>
+					<p>Curabitur nunc dolor, molestie efficitur ullamcorper posuere, eleifend vitae lorem.</p>
+				</details>
+			</li>
+			<li>
+				<h5>Header for step 4</h5>
+				<p>Quisque id accumsan tellus. Nunc sed volutpat ligula, eget semper arcu. Cras bibendum leo leo, sit amet varius turpis fermentum vel. Phasellus dapibus, ipsum sed volutpat efficitur, est massa dapibus nunc, vel vestibulum mauris nisl ac elit. Aliquam ultricies dignissim ligula ac interdum. Curabitur finibus cursus elementum. Maecenas ullamcorper interdum viverra. Sed massa eros, sagittis ut ipsum eget, lobortis pharetra leo.</p>
+			</li>
+		</ol>
+		<details>
+			<summary>View code</summary>
```

Comment:
Considering that the previous list do include inline heading and considering that a list is not an implicit section, the page structure need to be updated, like we can an heading here or we can change where the section are closing.


```suggestion
			<summary><h4>View code</h4></summary>
```